common/src/main/java/jp/co/soramitsu/common/view/SegmentedButtonView.kt

```diff
@@ -0,0 +1,99 @@
+package jp.co.soramitsu.common.view
+
+import android.content.Context
+import android.os.Parcelable
+import android.util.AttributeSet
+import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import jp.co.soramitsu.common.R
+import kotlinx.android.parcel.Parcelize
+import kotlinx.android.synthetic.main.view_segmented_button.view.optionOne
+import kotlinx.android.synthetic.main.view_segmented_button.view.optionTwo
+import kotlinx.android.synthetic.main.view_segmented_button.view.selectedBackground
+
+class SegmentedButtonView @JvmOverloads constructor(
```

Comment:
Button might have more than two options, please consider refactoring

Reply:
should we do this now or wait until we got button with more options? making generic SegmentedButton could be time consuming

Reply:
should we do this now or wait until we got button with more options? making generic SegmentedButton could be time consuming